### DISCUSSION TITLE
New to Plex and audiobooks, followed all instructions but nothing is showing up in Plex

### DISCUSSION THREAD

**Discussion by RHill051** [seanap/Plex-Audiobook-Guide#42] [Q&A]
Hey guys, I hope this is an acceptable place to ask for help. I'm by no means an expert when it comes to Linux but have a degree in software development so I'm relatively confident I've followed all the steps correctly but nothing is showing up in Plex so far. I've tried all the simple fixes like restarting the plex container in my UNRAID server, as well as rereading all the instructions and ensuring everything is up to date by executing the 'git pull' for the AudioNexus.bundle. The first audiobooks I'm trying to add to my collection are the chronicles of Narnia however the files I have been originally in MP4 format so I converted them to M4V before I processed them through MP3Tag which seemed to go fine. I have a hunch it could be the files I'm trying to use so in the meantime I'm going to try to add a different audio book but if not I'd greatly appreciate any help you guys can share on what I'm missing

**Comment by RHill051** [comment]
Well if by chance someone else has a similar issue and they are only testing with a single audio book (or set) I advise trying another audio book. I'm seeing the data show up fine so I guess I did follow everything correclty. :)

**Reply by seanap** [reply]
Interesting, glad you got it working. At first I was wondering if you had a permissions issue.  Plex supports the following formats for a music type library: 

- aac
- alac
- e-ac3
- flac
- mp3
- m4a
- m4b
- wav

I convert everything to a chapterized single file m4b.
